pkg/runtime/wfengine/workflowstate.go

```diff
@@ -256,7 +305,7 @@ func (s *workflowState) GetPurgeRequest(actorID string) (*actors.TransactionalRe
 	req := &actors.TransactionalRequest{
 		ActorType:  s.config.workflowActorType,
 		ActorID:    actorID,
-		Operations: make([]actors.TransactionalOperation, 0, 100),
+		Operations: make([]actors.TransactionalOperation, 0, len(s.Inbox)+len(s.History)+2),
```

Comment:
Add a comment stating what the +2 is for. I assume CustomStatus + MetaData?

Reply:
```suggestion
		// Initial capacity should be enough to contain the entire inbox, history, and custom status + metadata
		Operations: make([]actors.TransactionalOperation, 0, len(s.Inbox)+len(s.History)+2),
```